compose/compose-tooling/api/compose-tooling.api

```diff
@@ -5,6 +5,13 @@ public final class com/squareup/workflow/ui/compose/tooling/BuildConfig {
 	public fun <init> ()V
 }
 
+public final class com/squareup/workflow/ui/compose/tooling/ComposableSingletons$ViewFactoriesKt {
+	public static final field INSTANCE Lcom/squareup/workflow/ui/compose/tooling/ComposableSingletons$ViewFactoriesKt;
+	public static field lambda-1 Lkotlin/jvm/functions/Function4;
+	public fun <init> ()V
+	public final fun getLambda-1$compose_tooling_release ()Lkotlin/jvm/functions/Function4;
+}
+
```

Comment:
This is gross, but actually not the first occurance of this codegen pattern in the repo. I am surprised to see this pop up after only changing preview code though. Asked in [Kotlin Slack](https://kotlinlang.slack.com/archives/CJLTWPH7S/p1624299523093900).